build.gradle.kts

```diff
@@ -188,35 +189,44 @@ tasks.register<Sync>("resolve-default") {
     into(layout.buildDirectory.dir("${project.properties["lib.dir"]}"))
 }
 
+tasks.register<Sync>("resolve-test") {
+    description = "Resolve and retrieve dependencies"
+    dependsOn("clean-test-lib","init-nutch","copy-libs")
+    from(configurations.testCompileClasspath)
+    from(configurations.testRuntimeClasspath)
+    into(layout.buildDirectory.dir("${project.properties["test.lib.dir"]}"))
+}
+
 tasks.register("compile") {
     description = "Compile all Java files"
     dependsOn("compile-core","compile-plugins")
 }
 
 tasks.register<JavaCompile>("compile-core") {
     description = "Compile core Java files only"
-    dependsOn("init-nutch","resolve-default")
+    dependsOn("init-nutch","resolve-default","compileJava")
 
-    source = fileTree(layout.buildDirectory.dir("${project.properties["src.dir"]}"))
+    source = fileTree("${project.properties["src.dir"]}")
     include("org/apache/nutch/**/*.java")
-    destinationDirectory.set(layout.buildDirectory.dir("${project.properties["build.classes"]}"))
+    destinationDirectory.set(layout.projectDirectory.dir("${project.properties["build.classes"]}"))
     classpath = classpathCollection
     sourceCompatibility = "${project.properties["javac.version"]}"
     targetCompatibility = "${project.properties["javac.version"]}"
     
     options.annotationProcessorPath = classpathCollection
     options.sourcepath = layout.files("${project.properties["src.dir"]}")
-    options.compilerArgs.add("-Xlint-path")
+    options.compilerArgs.add("-Xlint:-path")
     options.encoding = "${project.properties["build.encoding"]}"
     options.isDebug = "${project.properties["javac.debug"]}" == "on"
     options.isDeprecation = "${project.properties["javac.deprecation"]}" == "on"
-
+    
     copy {
-        from(layout.buildDirectory.dir("${project.properties["src.dir"]}"))
-        include("**/*.html")
-        include("**/*.css")
-        include("**/*.properties")
-        into(layout.buildDirectory.dir("${project.properties["build.classes"]}"))
+        from(layout.projectDirectory.dir("${project.properties["src.dir"]}"))
+        include("**/*.html", "**/*.css", "**/*.properties")
```

Comment:
Please remove this line. [overview.html](https://github.com/csci401-apache-nutch/nutch/blob/NUTCH-2940/src/java/overview.html) is still not being copied over to `build/classes/overview.html`